src/main/java/rs/hybridit/controller/BookController.java

```diff
@@ -0,0 +1,13 @@
+package rs.hybridit.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RestController;
+import rs.hybridit.service.BookService;
+import rs.hybridit.serviceImpl.BookServiceImpl;
+
+@RestController
+public class BookController {
+
+    @Autowired
+    BookService bookService;
```

Comment:
use constructor DI, please read this:
https://dzone.com/articles/spring-di-patterns-the-good-the-bad-and-the-ugly